build-tools/installers/create-installers.targets

```diff
@@ -388,14 +388,18 @@
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)Xamarin.Installer.Common.dll" />
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)Xamarin.Installer.Common.pdb" />
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)Xamarin.Installer.Common.targets" />
-    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MicrosoftAndroidSdkOutDir)lib\%(Identity)\xamarin.sync')" />
-    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MicrosoftAndroidSdkOutDir)lib\%(Identity)\xamarin.find')" />
-    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MicrosoftAndroidSdkOutDir)lib\%(Identity)\xamarin.stat')" />
     <LegacyTargetsFiles Include="$(XAInstallPrefix)xbuild\Novell\Xamarin.Android.Bindings.targets" />
     <LegacyTargetsFiles Include="$(XAInstallPrefix)xbuild\Novell\Xamarin.Android.VisualBasic.targets" />
   </ItemGroup>
+  <Target Name="_FindFastDevTools" Condition=" '$(_HasCommercialFiles)' == 'True' ">
+    <ItemGroup>
+      <_FastDevNativeToolPaths Include="@(AndroidSupportedTargetJitAbi->'$(MicrosoftAndroidSdkOutDir)lib\%(Identity)\')" />
+      <_FastDevNativeTools Include="%(_FastDevNativeToolPaths.Identity)\xamarin.*" />
```

Comment:
We have fast dev tests. If any of these tools are missing we will fail.

Reply:
If possible, I would prefer to keep these files listed individually / explicitly to keep with the original intent of this file. We should fail early with a clear message when something we know we need to ship is missing.

Reply:
I'll figure out a way to drive this file list from the monodroid side. When we add new files its a pain since it requires bumps in both repo's before we can even merge the PR's

Reply:
I think this will work https://github.com/xamarin/monodroid/pull/1363